playbooks/roles/mattermail/templates/config.json.j2

```diff
@@ -0,0 +1,22 @@
+{
+    "Directory": "{{ MATTERMAIL_HOME }}/data/",
+    "Profiles": [
```

Comment:
So we don't seem to have a way to actually filter the things we want, because the `Filter` key is not present. See https://github.com/rodcorsi/mattermail#filter.

Additionally, I think we need to use the `UseAPIv3` key here from https://github.com/rodcorsi/mattermail#mattermost.

Please check that README to see if anything else we may need has been missed.

Reply:
Yeah, I expected a mail account getting only urgent emails, but filtering is available if that's not the case. There's also a redirect by subject option but there's little documentation on that.

I agree that UseAPIV3 might be necessary, let me know if you want me to make the change @xitij2000

Reply:
@UmanShahzad  @viadanna  I don't see why ``UseAPIV3`` is necessary at all. We are already using Mattermost 4.x. 

Am I missing something here?